packages/esm-patient-list-app/src/patient-table/patient-table.component.tsx

```diff
@@ -101,7 +109,7 @@ const PatientTable: React.FC<PatientTableProps> = ({
               id="patient-list-search"
               placeholder={search.placeHolder}
               labelText=""
-              size={isDesktop(layout) ? 'sm' : 'xl'}
+              size={isDesktop(layout) ? 'sm' : 'lg'}
```

Comment:
Hi @ayush-AI, Is this change required for the tests?

Reply:
```
 console.error
      Warning: Failed prop type: Invalid prop `size` of value `xl` supplied to `ForwardRef(Search)`, expected one of ["sm","md","lg"].
          at Search (/Users/ayushmishra/Desktop/openmrs/openmrs-esm-patient-management/node_modules/@carbon/react/lib/components/Search/next/Search.js:38:32)
          at patients (/Users/ayushmishra/Desktop/openmrs/openmrs-esm-patient-management/packages/esm-patient-list-app/src/patient-table/patient-table.component.tsx:57:3)
          at div
          at section
          at main
          at PatientListDetailComponent (/Users/ayushmishra/Desktop/openmrs/openmrs-esm-patient-management/packages/esm-patient-list-app/src/patient-list-detail/patient-list-detail.component.tsx:23:27)

      106 |         <div>
      107 |           <Layer>
    > 108 |             <Search
          |             ^
      109 |               id="patient-list-search"
      110 |               placeholder={search.placeHolder}
      111 |               labelText=""

      at printWarning (node_modules/react/cjs/react.development.js:209:30)
      at error (node_modules/react/cjs/react.development.js:183:7)
      at checkPropTypes (node_modules/react/cjs/react.development.js:2074:11)
      at validatePropTypes (node_modules/react/cjs/react.development.js:2275:7)
      at Object.createElementWithValidation [as createElement] (node_modules/react/cjs/react.development.js:2379:5)
      at PatientTable (packages/esm-patient-list-app/src/patient-table/patient-table.component.tsx:108:13)
      at renderWithHooks (node_modules/react-dom/cjs/react-dom.development.js:16305:18)
      at mountIndeterminateComponent (node_modules/react-dom/cjs/react-dom.development.js:20074:13)
   ```
   
   
 I was getting this error that's why I changed.